bluebrain/repo-bluebrain/packages/sim-model/package.py

```diff
@@ -223,6 +223,7 @@ def _setup_run_environment_common(self, env):
             env.set('HOC_LIBRARY_PATH', self.prefix.lib.hoc)
         if os.path.isdir(self.prefix.lib.python):
             env.prepend_path('PYTHONPATH', self.prefix.lib.python)
+        env.set('{}_ROOT'.format(self.name.upper()), self.prefix)
```

Comment:
IIUC this means things like `NEURODAMUS_THALAMUS_ROOT`.

Reply:
Yes, correct